.github/workflows/lint.yml

```
@@ -0,0 +1,21 @@
+name: Lint
+
+on: [push, pull_request]
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v3
+
+    - name: Set up Python 3.11
+      uses: actions/setup-python@v4
+      with:
+        python-version: '3.11'
```

We can set it to use a recent one instead so we don't need to bump it each year:

```suggestion
    - name: Set up Python
      uses: actions/setup-python@v4
      with:
        python-version: '3.x'
```